Makefile

```diff
@@ -415,3 +415,10 @@ show-container-engine:
 	@echo -n $(CONTAINER_ENGINE)
 
 
+imports: ## Organize imports in go files using goio. Example: make imports
```

Comment:
There is an existing `goimports` target (called as a dependency of `make fmt`). Is it redundant with this one?

Reply:
`gofmt` is a source code formatter, whereas `goio` is an imports organizer, they perform different functions, so no, it would not be redundant.

Reply:
Yes, but I was talking about `goimports` (from [golang.org/x/tools/cmd/goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)): https://github.com/janus-idp/operator/blob/main/Makefile#L269-L272
It also organizes the Go import lines (while also formatting the code as `gofmt`) So I was wondering if this should be removed as well..

Reply:
Done.